mm/2s2h/Rando/MiscBehavior/CheckQueue.cpp

```diff
@@ -101,3 +101,64 @@ void Rando::MiscBehavior::CheckQueueReset() {
     GameInteractor::Instance->currentEvent = GIEventNone{};
     GameInteractor::Instance->events.clear();
 }
+
+/// @brief Should only be used for non-unique items like junk or refills.
+/// @param itemId the item ID to attempt to give to the player
+void Rando::MiscBehavior::QueueNonCheckItem(RandoItemId itemId) {
```

Comment:
We might re-introduce something like this later on, but for now lets nix it in favor of my other proposed changes